bin/o2a-run-sys-test

```diff
@@ -186,6 +188,9 @@ Flags:
 -v, --verbose
         Add even more verbosity when running the script.
 
+-d, --dot
+        Creates files in the DOT representation.
+        If you have the imgcat program in PATH, the files will also be displayed in the console
```

Comment:
"if you have graphviz installed, the image will be converted to .png and stored in output and in case you have imgcat it will also be displayed in the console"

Reply:
I use a bash pipe, so an image file is not created.  Do you think that it is worth expanding this functionality?

Reply:
I updated the PR. Thanks.